packages/addons/addon-depends/enet/package.mk

```diff
@@ -2,8 +2,8 @@
 # Copyright (C) 2016-present Team LibreELEC (https://libreelec.tv)
 
 PKG_NAME="enet"
-PKG_VERSION="d9e561938fd9360cdbbd67d78b105ccbe4af0a65" # 10 Jan 2021 # 1.3.17+vita
-PKG_SHA256="ff52ea54edb71662d5933b165c073f079c90ed9adcf98bcb7b2e74d4ddf3dc6b"
+PKG_VERSION="8d69c5a" # 12 Jun 2021
```

Comment:
It's now standard to use full git sha not the short one. I suggest to use that too for consistency.